### PR TITLE
KAS-1232 Removing OC from the frontend

### PR DESCRIPTION
**Pull request by ValenberghsSven**

### CODE REVIEW

File: app/components/subcases/subcase-description/template.hbs [hunk 1]
```diff
@@ -48,62 +44,60 @@
           {{t "number-session"}}
         </span>
         <span class="vl-description-data__value">
-          {{#if (not (await subcase.isOC))}}
+          {{#if (await subcase.case.latestMeeting.number)}}
+            {{await subcase.case.latestMeeting.number}}
+          {{else}}
```
Comment (rafael-mindreau) [inline app/components/subcases/subcase-description/template.hbs]: I can't seem to figure out where this comes from?

File: app/controllers/application.js [hunk 2]
```diff
@@ -5,7 +5,6 @@ import { on } from '@ember/object/evented';
 import { inject } from '@ember/service';
 import isAuthenticatedMixin from 'fe-redpencil/mixins/is-authenticated-mixin';
 import { A } from '@ember/array';
-import moment from 'moment';
```
Comment (rafael-mindreau) [inline app/controllers/application.js]: I suppose moment was just simply unused here?

Comment (ValenberghsSven) [reply app/controllers/application.js]: Yes, unused import

File: app/components/subcases/subcase-description/template.hbs (hunk 1 above)
Comment (ValenberghsSven) [reply app/components/subcases/subcase-description/template.hbs]: this span used to be 
```
        <span class="vl-description-data__value">
          {{#if (not (await subcase.isOC))}}
            {{t "no-number-yet"}}
          {{else}}
            {{await subcase.case.OCMeetingNumber}}
          {{/if}}
        </span>
```

-subcase.isOC relation has been removed
-OCMeetingNumber was a computed value that was equal to `subcase.case.latestMeeting.number`
The computed was removed but the number value (if present) should still be shown on the subcase description view

**Pull request merged**